ci/federalist-pipeline.yml

```diff
@@ -0,0 +1,279 @@
+          CF_MANIFEST: .cloudgov/manifest.yml
+          CF_VARS_FILE: .cloudgov/vars/production.yml
+        on_failure:
+          try:
+            task: cancel-api-deployment
+            config:
+              <<: *cf-image
+              inputs: [name: src]
+              run:
+                dir: src
+                path: ci/tasks/cancel-deployment.sh
+            params:
+              <<: *env-cf
+              CF_APP_NAME: federalist-builder
+    on_failure:
+      in_parallel:
+        - put: gh-status
+          inputs: [src]
+          params: {state: failure}
+        - put: slack
+          params:
+            text: |
+              :x: FAILED: federalist-builder deployment on production
+              <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+            channel: ((slack-channel))
+            username: ((slack-username))
+            icon_url: ((slack-icon-url))
+    on_success:
+      in_parallel:
+        - put: gh-status
+          inputs: [src]
+          params: {state: success}
+        - put: slack
+          params:
+            text: |
+              :white_check_mark: SUCCESS: Successfully deployed federalist-builder on production
+              <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+            channel: ((slack-channel))
+            username: ((slack-username))
+            icon_url: ((slack-icon-url))
+
+  - name: nightly-tasks-production
+    plan:
+      - get: src
+        resource: src-production
+        passed: [set-pipeline]
+        params: {depth: 1}
+      - get: nightly
+        trigger: true
+      - task: restage
+        config:
+          <<: *cf-image
+          inputs: [name: src]
+          run:
+            dir: src
+            path: ci/tasks/restage.sh
+        params:
+          <<: *env-cf
+          CF_APP_NAME: federalist-builder
+
+############################
+#  RESOURCES
+
+resources:
+
+  - name: src-production
+    type: git
+    icon: github
+    source:
+      uri: ((git-base-url))/((builder-repository-path))
+      branch: main
+
+  - name: pr-main
+    type: pull-request
+    check_every: 1m
+    source:
+      repository: ((builder-repository-path))
+      access_token: ((gh-access-token))
+      base_branch: main
+      disable_forks: true
+      ignore_drafts: false
+
+  - name: nightly
+    type: time
+    source:
+      start: 12:00 AM
+      stop: 1:00 AM
+      location: America/New_York
+
+  - name: nginx
+    type: docker-image
+    source:
+      repository: nginx
+      tag: 1
+
+  - name: node
+    type: docker-image
+    source:
+      repository: node
+      tag: 12
+
+  - name: slack
+    type: slack-notification
+    source:
+      url: ((slack-webhook-url))
+
+  - name: gh-status
+    type: cogito
+    check_every: 1h
+    source:
+      owner: 18F
```

Comment:
I just realized we hardcoded this for all the pipelines so far. We may want to break this into variables to make transition of GH org simpler.